cmd/openfga/main_test.go

```diff
@@ -160,7 +161,7 @@ func createGrpcConnection(t *testing.T, tester OpenFGATester) *grpc.ClientConn {
 	conn, err := grpc.Dial(
 		tester.GetGRPCAddress(),
 		[]grpc.DialOption{
-			grpc.WithBlock(),
+			grpc.WithConnectParams(grpc.ConnectParams{Backoff: grpcbackoff.DefaultConfig}),
```

Comment:
Since this is shared in a lot of places, can we please put a list of the common dial opts in `pkg/testutils` or somewhere appropriate like that, and then reference that instead of constructing it in each place where we use the same value?

Reply:
Created one 👍